app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -1171,4 +1173,23 @@ class PatientRepositoryAndroidTest {
 
     assertThat(savedPatientProfiles).isEqualTo(expectedPatientProfiles)
   }
+
+  @Test
+  fun if_sync_is_pending_for_any_patient_record_then_it_should_be_counted_in_pendingRecordsCount() {
+
+    fun createPatientProfile(status: SyncStatus): PatientProfile {
+      return testData.patientProfile()
```

Comment:
`TestData#patientProfile` has a parameter for the `syncStatus`, so you do not need to manually copy the patient to change the sync status.

Reply:
Done